app/views/components/input_textarea.scala.html

```diff
@@ -29,6 +30,9 @@
         @hint.map { hint =>
             <span class="form-hint">@hint</span>
         }
+        @htmlHint.map { htmlHint =>
+            <span class="form-hint">@htmlHint</span>
```

Comment:
Minor, but is a span the right html element here? Looks more like a `div` scenario to me

Reply:
You have a good point.  I'll change